docs/gettingstarted/query_test_report.rst

```diff
@@ -20,9 +20,14 @@ with default format fields. To see a list of all format fields, click :ref:`here
 .. note::
    ``buildtest rt`` is an alias for ``buildtest report`` command.
 
-Color Options (``buildtest report --color``)
+Color output of buildtest report (buildtest report --color)
 ---------------------------------------------
 
+If you would like to change the output color for either ``buildtest report`` and ``buildtest report summary`` you can use the ``--color`` option.
+This will print the output of the previous two commands in the provided color. The color must be one of the supported colors by rich console.
+
+.. command-output:: buildtest report --color red 
+.. command-output:: buildtest report --color red summary
```

Comment:
lets not run these commands because readthedocs wouldnt render output of color in docs see https://buildtest--1207.org.readthedocs.build/en/1207/gettingstarted/query_test_report.html#color-output which is the build of this docs for this PR.

Instead let's put this as a block of code that is highlighted using the `::`` with tab indentation so it would be something like this


```
The color must be one of the supported colors by rich console, show below is an example of coloring output of tables with red::

    $ buildtest report --color red
    $ buildtest report --color red summary --detailed
```

We need to use `--detailed` otherwise the output doesnt color since default output of `buildtest rt summary` is just text and not table.

Reply:
I think that this is the way that yout want it let me know

Reply:
No if you use `.. command-output::` it will run the command and that will not work correctly. Just add this as a blob of text tab indented as i shared in https://github.com/buildtesters/buildtest/pull/1207#discussion_r966112535